Source/JavaScriptCore/runtime/PropertyTable.h

```diff
@@ -59,11 +59,6 @@ JS_EXPORT_PRIVATE extern PropertyTableStats* propertyTableStats;
 
 #endif
 
-inline constexpr bool isPowerOf2(unsigned v)
-{
-    return hasOneBitSet(v);
```

Comment:
Is the `hasOneBitSet` function still being used?

Reply:
Yes, it is still being used a lot. I am planning to follow-up though since `std::has_single_bit()` exists in C++20. That said, we have the issue that `hasOneBitSet()` can be called with signed types while `std::has_single_bit()` cannot.